src/js/update-sites.js

```diff
@@ -20,35 +19,49 @@ let vm = new Vue({
     packages: [],
     codes: [],
     sites: [],
-    siteQuery: []
+    siteQuery: '?where={"code.package":"5972304889b0dc0c61cf5df3"}'
   },
   methods: {
     getSites(siteQuery) {
       let that = this;
-      let endpoint = '/sites';
-
-      atlasRequest(baseURL, endpoint, query = siteQuery, method = 'GET', body = null, etag = null).then(data => that.sites = data);
+      let endpoint = 'sites';
+      console.log(packageBaseURL+endpoint+siteQuery);
+
+      atlasRequest(packageBaseURL, endpoint, query = siteQuery, method = 'GET', body = null, etag = null).then(data => that.sites = data[0]).then(
+        function setPackages() {
+          let singlePackage = [];
+          for(site of that.sites) {
+            for(single of site.code.package) {
+              singlePackage.push(single);
+            }
+          }
+          var names = singlePackage;
+          var uniqueNames = [];
+          $.each(names, function(i, el){
```

Comment:
I think `$` is jQuery? If this is just a foreach loop then you should be able to do `names.forEach(function(i,el){...});`